content/manuals.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Manuals
+description: Explore the Docker product user manuals
+keywords: docker, docs, manuals, products, user guides, how-to
+notoc: true
+skip_read_time: true
+development:
+- title: Docker Desktop
+  description: Your command center for container development.
+  icon: /assets/icons/Whale.svg
+  link: /desktop/
+- title: Docker Compose
+  description: Define and run multi-container applications.
+  icon: /assets/icons/Compose.svg
+  link: /compose/
+- title: Docker Build
+  description: Build and ship any application anywhere.
+  icon: build
+  link: /build/
+- title: Docker Engine
+  description: The industry-leading container runtime.
+  icon: developer_board
+  link: /engine/
+- title: Docker Extensions
+  description: Customize your Docker Desktop workflow.
+  icon: extension
+  link: /desktop/extensions/
+services:
+- title: Docker Hub
+  description: Discover, share, and integrate container images
+  icon: hub
+  link: /docker-hub/
+- title: Docker Scout
+  description: Image analysis and policy evaluation.
+  icon: /assets/icons/Scout.svg
+  link: /scout/
+- title: Trusted content
+  description: Curated, high-quality content from trusted sources.
+  icon: /assets/icons/Verified.svg
+  link: /trusted-content/
+admin:
+- title: Administration
+  description: Centralized observability for companies and organizations.
+  icon: admin_panel_settings
+  link: /admin/
+- title: Security
+  description: Security guardrails for both administrators and developers.
+  icon: lock
+  link: /security/
+- title: Billing
+  description: Manage billing and payment methods.
+  icon: payments
+  link: /billing/
+- title: Subscription
+  description: Commercial use licenses for Docker products.
+  icon: card_membership
+  link: /subscription/
+---
+
+This section contains user guides on how to install, set up, configure, and use
+Docker products.
```

Comment:
As we've had some user confusion around what goes into Manuals v/s Guides, suggest we add a link to the guides from here for users who'd like to get started with Docker

Reply:
I don't know, it feels wrong to link off to a completely different section the first thing we do on this landing page.

I would rather we release it like this. The landing page for the guides section should clear this up. They're all linked on the landing page and in the site header.